src/panel/pets.ts

```diff
@@ -800,7 +800,7 @@ export class Clippy extends BasePetType {
     }
     hello(): string {
         // TODO: #188 Add a custom message for clippy
-        return ` says hello 👋!`;
```

Comment:
Please remove the line above with the todo as this will be closed with the merged PR

Reply:
I have removed the comment @tonybaloney.